get_cpu_process/cpu

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+from sys import argv
+from sys import exit
+from get_cpu_process import get_pid
+from get_cpu_process import get_process_run_time
+from get_cpu_process import is_running
+
+
+def help():
+    print("")
+    print("Simple CPU related script written by Zac the Wise utilising psutil")
+    print("")
+    print("Basic usage: cpu \"<process-name>\"")
+    print("")
+    print("Example: cpu \"amethyst\"")
+    print("Returns 'running' or 'not running'")
+    print("")
+    print("")
+    print("Advanced usage: cpu <option> <arguement>")
+    print("")
+    print("     --get-pid \"<process-name>\"         returns the process id")
+    print("")
+    print("     --run-time \"<process-name>\">       returns the process running-time")
+    print("")
```

Comment:
I would use multilingual strings, then it would be possible to beautifully describe the entire text and would not have to escape quotes
However, if you use https://docs.python.org/3/library/argparse.html#example then this is not needed - it generates help text

Reply:
Done 👍. First time learning and using argparse so pls let me know if I can improve anything